src/schemas/cart.js

```diff
@@ -0,0 +1,14 @@
+var apollo = require('apollo-server-express');
```

Comment:
It's a pretty small detail but you're not interested in using anything inside the apollo namespace here, only `gql` so you could instead do this:

```javascript
const { gql } = require('apollo-server-express');
```

Same applies to everywhere else.

Reply:
Way better. ths